index.js

```diff
@@ -40,11 +36,10 @@ function pixelHighlight(id) {
 
     // Creates List of Pixels
     let previousPixels = document.querySelectorAll('.pixel')
-    function resetBorders() {
-        for (var x = 0; x < previousPixels.length; x++) {
-            previousPixels[x].style.border = '0px solid gray' 
-        }
-    } resetBorders()
```

Comment:
I don't see a reason to nest this logic in an inner function. 

Typically we use inner functions for situations where:
- we're reusing the same functionality over and over for some reason 
- the arguments to that functionality might change
- if it's an interior function (rather than one we move out to be in a neighboring scope, it's probably because there are a bunch of arguments that would otherwise need to be passed into the function that we can use for the enclosing scope

```
// Interior function 
func a() {
  const a, b, c = 1, 2, 3
  func b_interior() { console.log(a,b,c) }
  b_interior()
}


// Neighbor function
func b_neighbor(a, b, c) { console.log(a,b,c) }

func a() {
  const a, b, c = 1, 2, 3
  func b_neighbor(a, b, c)
}

```

Reply:
This makes sense.  I think I was trying to make it re-usable by writing it this way, but this isn't a function that needs to be re-used that way.